hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,277 @@
+(via chat).
+
+Note: This feature requires server-side support and is thus only available on servers at least running Mumble 1.4.0.
+
+![Channel listeners](media/ChannelListeners.png)
+
+
+### TalkingUI
+
+Have you ever been in a situation where you heard someone talking but couldn't quite figure out who it was? Then this feature is probably for you!
+
+The TalkingUI can be considered the equivalent of the already existing overlay but for non-gamers or in other terms "for everyday usage".
+
+This feature adds an optional floating window containing a list of currently speaking users and their respective channels. This allows you to quickly
+see who's currently talking and in which channel that person currently is in (this becomes important if you want to respond to that person).
+
+The big difference between the TalkingUI and Mumble's main window (which of course also contains this information) is that the TalkingUI does not
+contain all the other UI elements nor does it necessarily show the entire channel-tree. Furthermore it automatically resizes itself to the minimal
+size needed to display the respective information. Thus it usually does not take up a lot of space and can therefore be left floating in the corner of
+the screen without consuming too much screen real estate.
+
+Note that the TalkingUI is highly configurable (Settings -> User Interface) and also does provides selection and context-menu support just as you get
+it in the main window. In fact the selection between TalkingUI and main window is synchronized and thus this plays really nice with the new "whisper
+to selected" feature allowing to whisper to whatever currently is selected in the UI (be it a channel or a user).
+
+![TalkingUI](media/TalkingUI.png)
+
+
+### UI indication for access restrictions
+
+In previous versions of Mumble there was no indication of whether you are able to join a channel or not. You just had to try it out and either it
+worked or you got a Permission Denied message. This feature solves this problem by marking "access-restricted" channels with a lock icon. Access
+restricted channels are all channels that are either only open to a specific group on the server or require a password (access token) to join.
+
+In total a channel can be in one of three states: Either it is not access-restricted in which case it just looks like channels used to look prior to
+the introduction of this feature or it is indeed access-restricted. In the latter case a lock icon is shown to the right of channel name. This look
+can either be open and green or closed and red. Green means that you can enter the channel nonetheless (you belong to the required group or have the
+correct password (access token) set) whereas red means that you are not allowed to enter the channel.
+
+Note that this is also a feature that requires server-side support and thus only takes effect on servers that are at least running Mumble version
+1.4.0.
```

Comment:
```suggestion
Note that this is also a feature that requires server-side support and thus only takes effect on servers that are at least 1.4.0.
```

Reply:
What is this change about? removing a newline? if so: why?

Reply:
I changed the wording.

Reply:
Ah I see. Same change as above :bulb: